demo/src/index.js

```diff
@@ -241,6 +241,9 @@ module.exports = {
       // wrapperScreens
       get TouchableOpacityScreen() {
         return require('./screens/componentScreens/TouchableOpacityScreen').default;
+      },
+      get ProductPage() {
+        return require('./screens/realExamples/ProductPage');
```

Comment:
Let's move this export to be part of `// realExamples` 
just above `// wrapperScreens`